docs/components/table.md

```diff
@@ -474,6 +475,30 @@ Example headers array:
 </script>
 ```
 
+#### `allowHide`
```

Comment:
```suggestion
#### allowHide
```

nit: I'm trying to bring all headers to one standard which is not to apply any styling to header text of any level

Reply:
See my comment on the prop naming 😄